packages/fuels/tests/providers.rs

```diff
@@ -194,6 +194,7 @@ async fn test_input_message_pays_fee() -> Result<()> {
 }
 
 #[tokio::test]
+#[ignore]
```

Comment:
Ignore these now because you can't use `manual_blocks_enabled` while using PoA. Once we bring it back to `Instant`, we'll re-enable those.